tools/logger/logger.c

```diff
@@ -247,6 +247,7 @@ int main(int argc, char *argv[])
 				usage();
 			}
 			config.dump_ldc = 1;
+			config.version_fw = 0;
```

Comment:
typo in the commit message and can you state why it make no sense.

Reply:
I didn't spot typo, but message is already extended